keyboards/ai03/polaris/polaris.h

```diff
@@ -40,7 +40,7 @@
 	{ K400,  K401,  K402,  KC_NO, K404,  KC_NO, K406,  KC_NO, K408,  KC_NO, K410,  K411,  K412,  K413 }  \
 }
 
-#define LAYOUT_60_ansi_tsangan_split_rshift( \
+#define LAYOUT_60_tsangan_hhkb( \
```

Comment:
To maintain backwards compatibility, you will need to also add
```c
#define LAYOUT_60_ansi_tsangan_split_rshift LAYOUT_60_tsangan_hhkb
```
to the bottom of the file

Reply:
I'm happy to do that if you want, but does it matter in this case since this board was just checked in yesterday and the hardware hasn't shipped out yet?

Reply:
This is partially my fault; I forgot `60_tsangan_hhkb` was a thing.

IMO it doesn't matter, due to the recency of this board's addition.

Reply:
Thats fine with me.